tests/JWT.Tests.Net35/JWT.Tests.Net35.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
It can't be 35? If we're adding a separate project, then let's to use the same "low" bar

Reply:
MSTest V2 does not support net35. I spent time looking for alternatives but they all ran into a dead end. Switching unit testing approach seems not worth the effort in this case. Therefore, the next best thing I went for is to test the net35 code on the net46 runtime. Lower than net46 causes NuGet dependency conflicts.

Reply:
I see, makes sense. But should we use the latest .NET Framework in this project too then?

Another idea to have Net35, Net45, Net462, Net472, Net48, etc? Or it doesn't make sense to have more than 2?

Reply:
Given that installations of .Net Framework 4.5 and upwards are updated in place when newer versions come out, you may as well test everything on the same target framework, in which case I would go for the lowest version still supported by the required NuGet dependencies.

Reply:
Ok, sounds good